phpunit.xml.dist

```diff
@@ -16,8 +16,10 @@
     convertNoticesToExceptions="true"
     convertWarningsToExceptions="true"
     enforceTimeLimit="true"
+    timeoutForSmallTests="2"
     processIsolation="false"
     stopOnFailure="false"
+    failOnRisky="false"
```

Comment:
This changed to a default `true` in PHPUnit 9.x.

Reply:
what risky tests we have? maybe we should keep `true` and fix risky tests?

Reply:
I think I fixed all of them, but let's see what CI would say.

Reply:
CI seems to be happy.